docs/user/data.rst

```diff
@@ -558,7 +558,8 @@ Basic Usage
 
 ``async_mmap`` should be used only as formal parameters in leaf-level tasks.
 It can be constructed from ``mmap``, and an ``mmap`` argument can be passed to
-an ``async_mmap`` parameter.
+an ``async_mmap`` parameter. Detailed usage can be found at
+:ref:`Efficient Memory Accesses <tutorial/async_mmap:Efficient Memory Accesses>`
 
 .. warning::
```

Comment:
please add a warning that async mmap must be used with non-blocking read and write